### ISSUE TITLE
Pickup objects looks ok for new games but are flawed for loaded games

### ISSUE THREAD

**Issue by Florin9doi** [rwengine/openrw#389]
![untitled](https://user-images.githubusercontent.com/2995486/35999634-2e0b2b26-0d29-11e8-8dd2-c84ad66b3e8f.png)

**Comment by ShFil119** [comment]
Did it work before #377? Maybe I've caused some regression. :(

**Comment by danhedron** [comment]
I don't think this ever worked because of how openrw differentiates between pickups and normal objects.

Some more details on how the save file identifies pickups are required to fix the openrw behaviour.
